### PR TITLE
Fixes #34796

### LINKED ISSUE #34796: With v22.0, [protobuf] have taken on an explicit dependency on [abseil]
https://github.com/protocolbuffers/protobuf/releases/tag/v22.0

### PR DESCRIPTION
**Pull request by olejniczak**
Fixes #34796

### CODE REVIEW

**Comment by jimwang118** [comment]
You need to run the command:`./vcpkg x-add-version {portname}` and submit the generated file.

**Comment by olejniczak** [comment]
I wanted to fix #34537 pull request. In my pr there is no version change.

**Comment by jimwang118** [comment]
> I wanted to fix #34537 pull request. In my pr there is no version change.



> I wanted to fix #34537 pull request. In my pr there is no version change.

The above command is to update the version number and commit record defined in `vcpkg`. You can refer to our [maintainer-guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md#versioning) on how to modify the version.

File: ports/protobuf/vcpkg.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "homepage": "https://github.com/protocolbuffers/protobuf",
```
Comment (jimwang118) [inline ports/protobuf/vcpkg.json:5]: You need to add the keyword "port-version": 1.

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

**Comment by olejniczak** [comment]
Thank you for your help. Yes, indeed, it was some recommendations, hopefully there are reasonable.

**Comment by dg0yt** [comment]
Please use a more telling PR title. It will become part of the git history.

**Pull request closed without merging**